dpdispatcher/contexts/openapi_context.py

```
@@ -23,6 +23,36 @@
 )
 
 
+def unzip_file(zip_file, out_dir="./"):
+    obj = ZipFile(zip_file, "r")
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)